src/index.json

```diff
@@ -298,6 +298,63 @@
                 "sha256Digest": "1faecf9d3b5955829e00b228e174fa6bf1ba327f0d64c2edfc9d096d2926e8dc"
             }
         ],
+        "azure-devops-extension": [
```

Comment:
Can you rename this to `azure-devops`? This being an extension of the CLI is implicit.